src/DependencyInjection/SetonoSyliusMeilisearchExtension.php

```diff
@@ -59,6 +63,14 @@ public function load(array $configs, ContainerBuilder $container): void
         $container->setParameter('setono_sylius_meilisearch.server.master_key', $config['server']['master_key']);
         $container->setParameter('setono_sylius_meilisearch.server.search_key', $config['server']['search_key']);
 
+        // cache
+        /** @var bool $metadataCacheEnabled */
+        $metadataCacheEnabled = $config['metadata']['cache'] ?? false;
```

Comment:
1. Psalm knows the type because of your phpdoc above
2. It's always set according to your `Configuration`

So you don't need the variable I believe